warehousecontroller/mission2goal-translation-design.txt

```diff
@@ -0,0 +1,13 @@
+Mission Goal:
+	[TYPE, any/blue/green/red, ALL/[num], CURRENT/shelf1/shelf2/.., cbOut1/cbOut2/.]
+PDDL Goal:
+
+Num:CURRENT
+	Exists x1, .. , xn. AND_i ((type(xi) = T)(posn(xi) = cbOut2))
```

Comment:
Actually exists is allowed! I will try to define a Goal Definition shape, that can be used in the precondition and in the goal statement @swarupmohalik  we can have a followup on this on Monday.